luna/featurevis/transformations.py

```diff
@@ -0,0 +1,152 @@
+"""
+The transformations for feature vis process
+"""
+import random
+import tensorflow as tf
+import tensorflow_addons as tfa
+
+
+
+def add_noise(img, noise, pctg):
+    """Adds noise to the image to be manipulated.
+    Args:
+        img (list): the image data to which noise should be added
+        noise (boolean): whether noise should be added at all
+        pctg (number): how much noise should be added to the image
+        channels_first (bool, optional): whether the image is encoded channels
+        first. Defaults to False.
+    Returns:
+        list: the modified image
+    """
+    if noise:
+        if tf.compat.v1.keras.backend.image_data_format() == "channels_first":
+            img_noise = tf.random.uniform((img.shape),
+                                          dtype=tf.dtypes.float32)
+        else:
+            img_noise = tf.random.uniform((img.shape),
+                                          dtype=tf.dtypes.float32)
+        img_noise = (img_noise - 0.5) * 0.25 * ((100 - pctg) / 100)
+        img = img + img_noise
+        img = tf.clip_by_value(img, -1, 1)
+    return img
+
+
+def blur_image(img, blur, pctg):
+    """Gaussian blur the image to be modified.
+
+    Args:
+        img (list): the image to be blurred
+        blur (boolean): whether to blur the image
+        pctg (number): how much blur should be applied
```

Comment:
Again, we could remove `blur` and just check if `pctg` is `0`, no?

Reply:
Same as above